Implemented sorting and filtering for Activity page

bjorntp opened the pull request:
<!--
Ensure your title is short, descriptive, and in the imperative mood (Fix X, Change Y, instead of Fixed X, Changed Y).
For a good inspiration of what to write in commit messages and PRs please review https://chris.beams.io/posts/git-commit/ and our https://jellyfin.org/docs/general/contributing/issues page.
-->

**Changes**
- Added support for backend filters and sorting in Activity page.
- Changed filters for Severity and Name to single-select to comply with backend. 
- Added range support filter for dates
- Disabled the regular search bar as there is no search support in the backend. In my opinion it is redundant as well now that the filters work.

This relies on code from the unstable backend build. Keep that in mind if testing.

**Issues**
 #6221

bjorntp committed:
Fully implemented sorting and filtering

Note search and end date

bjorntp committed:
Disabled search bar and single time slot

commented out code related to search

bjorntp committed:
Added range support for dates and size modifications for columns

Date ranges should work once jellyfin/jellyfin/pull/15820 (approved)
have been merged. Works fine with local testing.

bjorntp committed:
Merge branch 'master' into 6221

bjorntp committed:
Default show filters

bjorntp committed:
Merge branch 'master' into 6221

bjorntp committed:
Contributors

bjorntp committed:
Merge branch 'master' into 6221

thornbill reviewed (changes requested):
1. You removed search param support so links from elsewhere in the dashboard are not behaving as expected.
2. The filter options should not be shown by default.
3. You shouldn't be making unrelated whitespace changes.

bjorntp committed:
PR-Comments

Reinstated search-params
Filters default to hidden
Reverted unrelated Prettier whitespace changes

bjorntp committed:
Merge branch '6221' of github.com:bjorntp/jellyfin-web into 6221

bjorntp committed:
PR-Comment

Actually fixed the formatting

bjorntp committed:
Merge branch 'master' into 6221

bjorntp commented:
FYI, I have made the changes. I just want to add the comment since I suspect that you get a lot of comments on GitHub so you might not see a request, and since you have a large workload here I do not want it to slip through the cracks.

thornbill committed:
Merge branch 'master' into 6221
